.github/workflows/release.yml

```diff
@@ -0,0 +1,93 @@
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+
+name: Release policy - test
+
+jobs:
+  test:
+    name: run tests and linters
+    uses: ./.github/workflows/reusable-test-policy-go.yml
+
+  release:
+    needs: test
+    permissions:
+      # Required to create GH releases
+      contents: write
+      # Required to push to GHCR
+      packages: write
+      # Required by cosign keyless signing
+      id-token: write
+
+    # TODO: go back to using the reusable workflow once we don't need the special tinygo build
```

Comment:
If tinygo 0.30.0 is broken, I wonder if we need to fix the reusable workflows for all policies, and if not, why is it broken only for this policy.

In any case, it may be good to consider exporting the tinygo version from reusable-build-go (already done) -> reusable-{test,release}-go (to be done) -> the workflow call in this repo.

Reply:
All the published versions of TinyGo are lacking this feature, this is however going to be part of the next version of tinygo (I've made sure of that).

I think can just sit and wait for this version to be out